web/src/main/java/com/graphhopper/http/DefaultModule.java

```diff
@@ -144,4 +169,38 @@ protected void configure() {
             throw new IllegalStateException("Couldn't load graph", ex);
         }
     }
+
+    public static class SpatialRuleListReflectionFactory extends SpatialRuleLookupBuilder.SpatialRuleListFactory {
```

Comment:
can we move this out of DefaultModule including the buildIndex method, which I only placed here as it was rather short. Now we should better separate this (?)